axis/resonant_axis.f95

```diff
@@ -0,0 +1,111 @@
+module resonant_axis
+
+use global_parameters
+implicit none
+
+contains
+
+!----------------------------------------------------------------------------------------------
+subroutine make_id_matrix_2body(pl_name,max_order)
```

Comment:
Renamed and moved to "id_matrix" module.
There are still questions about unit test...